src/clj/rems/config.clj

```diff
@@ -42,16 +44,20 @@
   (-> config
       (update :email-retry-period #(Period/parse %))
       (update-existing :extra-pages (partial mapv #(update-existing % :roles set)))
-      (update :disable-commands (partial mapv keyword))))
+      (update-existing :disable-commands #(->> (transform [ALL string?] keyword %)
+                                               (transform [ALL map? MAP-KEYS] keyword)
+                                               (transform [ALL map? MAP-VALS ALL] keyword)))))
```

Comment:
We could walk the `:disabled-commands` with a simple `for` since it's just a seq and do any necessary coercion and normalization here (without having to use Specter). If we normalized every item here, then the other code may become simpler.

Or this level of coercion could use Schema instead. This could provide some detailed validation message.